spec/factories/bulkrax_exporters.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+# Copied here from Bulkrax so they can be availabe for our tests
+# Later we could make a PR to expose them in Bulkrax so we don't have to copy them anymore
+
+FactoryBot.define do
+  factory :bulkrax_exporter, class: 'Bulkrax::Exporter' do
+    name { 'Export from Importer' }
```

Comment:
I've seen you've used double and single quotes. Let's keep Paul happy :)
```suggestion
    name { "Export from Importer" }
```